Mathlib/Analysis/NormedSpace/WeakDual.lean

```diff
@@ -29,15 +29,15 @@ topology.
 The main definitions concern the canonical mapping `Dual 𝕜 E → WeakDual 𝕜 E`.
 
 * `NormedSpace.Dual.toWeakDual` and `WeakDual.toNormedDual`: Linear equivalences from
-  `dual 𝕜 E` to `WeakDual 𝕜 E` and in the converse direction.
+  `ContinuousLinearMap.Dual 𝕜 E` to `WeakDual 𝕜 E` and in the converse direction.
 * `NormedSpace.Dual.continuousLinearMapToWeakDual`: A continuous linear mapping from
-  `Dual 𝕜 E` to `WeakDual 𝕜 E` (same as `NormedSpace.Dual.toWeakDual` but different bundled
-  data).
+  `ContinuousLinearMap.Dual 𝕜 E` to `WeakDual 𝕜 E` (same as `NormedSpace.Dual.toWeakDual`
+  but different bundled data).
 
 ## Main results
 
-The first main result concerns the comparison of the operator norm topology on `dual 𝕜 E` and the
-weak-* topology on (its type synonym) `WeakDual 𝕜 E`:
+The first main result concerns the comparison of the operator norm topology on
+`ContinuousLinearMap.Dual 𝕜 E` and the weak-* topology on (its type synonym) `WeakDual 𝕜 E`:
```

Comment:
```suggestion
```

This sentence in the module docstring seems to have become weird, due to the addition of Banach-Alaoglu. I suggest simply removing it.